phoenicis-dist/src/resources/Phoenicis.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
+Encoding=UTF-8
+Name=Phoenicis
```

Comment:
Shouldn't this be "Phoenicis PlayOnLinux" (after the discussion we had yesterday)?